code/mission/missionparse.cpp

```diff
@@ -424,11 +424,11 @@ void parse_custom_bitmap(const char *expected_string_640, const char *expected_s
 	// error testing
 	if (Fred_running && (found640) && !(found1024))
 	{
-		Warning(LOCATION, "Mission: found an entry for %s but not a corresponding entry for %s!", expected_string_640, expected_string_1024);
+		nprintf(("General", "Mission: found an entry for %s but not a corresponding entry for %s!", expected_string_640, expected_string_1024));
```

Comment:
Isn't `nprintf(("General"` the same as `mprintf((`? If that is the case then maybe this should use `mprintf` instead.

Reply:
I don't actually know.  There are plenty of `nprintf(("General"` statements throughout the codebase which have to do with parsing so that's why I used it.

Reply:
There is an `nprintf(("Parse"` but it's only used by `display_parse_diagnostics()`.  What do you think of changing this to `Parse` and then opening a separate PR to change all applicable `mprintf` statements?

Reply:
If it's a parse specific log message then "Parse" would be good too.

Reply:
It's not as cut-and-dried as I thought, so I left the other `mprintf`s alone.  I don't think it matters whether this one is `nprintf(("General"` or `mprintf((`, but I changed it.